eq-author/src/components/ContentPickerv2/DestinationPicker/Menu.test.js

```diff
@@ -126,131 +134,173 @@ function defaultSetup() {
+    beforeEach(() => {
+      questionnaire = { hub: false };
+      useQuestionnaire.mockImplementation(() => ({ questionnaire }));
+    });
 
-  it("should show active styling for current tab", () => {
-    const { getByText } = defaultSetup();
-    expect(getByText(tabTitles.current)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-  });
+    it("should default to current section tab", () => {
+      const { getByText } = defaultSetup();
+      expect(getByText("Question one")).toBeVisible();
+    });
 
-  it("should display question pages in 'Current section'", () => {
-    const { getByText } = defaultSetup();
-    expect(getByText(tabTitles.current)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-    expect(getByText("Question one")).toBeVisible();
-    expect(getByText("Question two")).toBeVisible();
-    expect(getByText("Question three")).toBeVisible();
-    expect(getByText("Question four")).toBeVisible();
-  });
+    it("should show active styling for current tab", () => {
+      const { getByText } = defaultSetup();
+      expect(getByText(tabTitles.current)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
+    });
 
-  it("should display sections in 'Later sections'", () => {
-    const { queryByText, getByText, click } = defaultSetup();
-    click(tabTitles.later);
-    expect(getByText(tabTitles.later)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-    expect(queryByText("Section one")).toBeFalsy();
-    expect(getByText("Section two")).toBeVisible();
-    expect(getByText("Section three")).toBeVisible();
-    expect(getByText("Section four")).toBeVisible();
-  });
+    it("should display question pages in 'Current section'", () => {
+      const { getByText } = defaultSetup();
+      expect(getByText(tabTitles.current)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
+      expect(getByText("Question one")).toBeVisible();
+      expect(getByText("Question two")).toBeVisible();
+      expect(getByText("Question three")).toBeVisible();
+      expect(getByText("Question four")).toBeVisible();
+    });
 
-  it("should display 'Other destinations' options", () => {
-    const { getByText, click } = defaultSetup();
-    click(tabTitles.other);
-    expect(getByText(tabTitles.other)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-    expect(getByText(destinationKey[NextPage])).toBeVisible();
-    expect(getByText(destinationKey[EndOfCurrentSection])).toBeVisible();
-    expect(getByText(destinationKey[EndOfQuestionnaire])).toBeVisible();
-  });
+    it("should display sections in 'Later sections'", () => {
+      const { queryByText, getByText, click } = defaultSetup();
+      click(tabTitles.later);
+      expect(getByText(tabTitles.later)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
```

Comment:
The same as above regarding testing the active indicator rather than the styles.